pyproject.toml

```diff
@@ -1,9 +1,62 @@
 [build-system]
 requires = [
-    "setuptools",
+    "setuptools>=61.2",
     "setuptools_scm>=6.2",
     "wheel",
     "jinja2>=2.10.3",
     "numpy>=2.0.0rc1",
 ]
 build-backend = 'setuptools.build_meta'
+
+[project]
+name = "pyerfa"
+authors = [{name = "The PyERFA Developers"}]
+license = {text = "BSD 3-Clause License"}
+description = "Python bindings for ERFA"
+readme = "README.rst"
+keywords = ["astronomy", "astrophysics", "cosmology", "space", "science", "coordinate"]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: C",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering :: Astronomy",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+urls = {Homepage = "https://github.com/liberfa/pyerfa"}
+requires-python = ">=3.9"
+dependencies = ["numpy>=1.19.3"]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+test = [
+    "pytest",
+    "pytest-doctestplus>=0.7",
+]
+docs = ["sphinx-astropy>=1.3"]
+
+[tool.setuptools]
+# We set packages to find: to automatically find all sub-packages
+zip-safe = false
+license-files = ["LICENSE.rst"]
+include-package-data = false
+
+[tool.setuptools.packages.find]
+exclude = ["erfa._dev"]
+namespaces = false
+
+[tool.pytest.ini_options]
+minversion = "4.6"
+testpaths = ['"erfa"', '"docs"', '"README.rst"']
+doctest_plus = "enabled"
+doctest_optionflags = """
+NORMALIZE_WHITESPACE
+ELLIPSIS
+FLOAT_CMP
+IGNORE_EXCEPTION_DETAIL"""
+text_file_format = "rst"
+addopts = "--doctest-rst"
+xfail_strict = true
+norecursedirs = ["'erfa/_dev'"]
```

Comment:
astropy again has just the double quote and `[\\/]` instead of just the `/` (presumably so things work on windows?).

Reply:
I don't think escaping is necessary. Pytest's doc is a bit terse but it does mention here and there that path should be written in UNIX style, so I figure it should be handling portability natively.

Reply:
Indeed, presumably our tests show this is working on windows... Thanks!